csrc/velox/functions/functions.h

```diff
@@ -262,6 +263,19 @@ inline void registerTorchArrowFunctions() {
       velox::ArrayWriterT<velox::Varchar>,
       std::shared_ptr<GPT2BPEEncoder>,
       velox::Varchar>({"bpe_tokenize"});
+
+    velox::registerFunction<
+      add_tokens,
+      velox::Array<int32_t>,
+      velox::Array<int32_t>,
+      velox::Array<int32_t>,
```

Comment:
maybe also try add one with int64_t? I remember python by default cast all the integer literals to int64.